components/automate-ui/src/app/entities/automate-settings/automate-settings.requests.ts

```diff
@@ -22,49 +25,111 @@ export class AutomateSettingsRequests {
   // fetchJobSchedulerStatus sends an HTTP GET Request to read the status of the JobScheduler
   // living inside the ingest-service
   public fetchJobSchedulerStatus(_params): Observable<JobSchedulerStatus> {
-    const url = `${RETENTION_URL}/nodes/status`;
+    // NEW ENDPOINT FOR DATA-LIFECYCLE
+    const url = '/api/v0/data-lifecycle/status';
```

Comment:
1. Probably do not need the comment there; that's only useful while reading this PR.
2. For the URL, we want to use the same convention as what you are supplanting: replace `/api/v0/data-lifecycle` with a value from the environment files. (In fact, if the name still makes sense, you could just update the definition of `RETENTION_URL` in the environment files and you would not need any changes in this file.